pkgs/development/python-modules/i3ipc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchurl
+, enum-compat
+, xorgserver, pytest, i3, python
+}:
+
+buildPythonPackage rec {
+  pname = "i3ipc";
+  version = "1.4.0";
+
+  src = fetchurl {
+    url = "${meta.homepage}/archive/v${version}.tar.gz";
+    sha256 = "06x1zc9qizmcg4lxlh2w5amcq7igqlrs1kkvvi1czb5wy689ygsi";
+  };
+
+  nativeBuildInputs = [ xorgserver pytest i3 ];
```

Comment:
This is probably better suited for `checkInputs`.

Reply:
fixed